framework/external/easylogging/easylogging++.cc

```diff
@@ -1786,7 +1786,7 @@ std::string TypedConfigurations::resolveFilename(const std::string& filename) {
         fmt = std::string(base::consts::kDefaultDateTimeFormatInFilename);
       }
       base::SubsecondPrecision ssPrec(3);
-      std::string now = base::utils::DateTime::getDateTime(fmt.c_str(), &ssPrec);
+      static std::string now = base::utils::DateTime::getDateTime(fmt.c_str(), &ssPrec);
       base::utils::Str::replaceAll(now, '/', '-'); // Replace path element since we are dealing with filename
```

Comment:
I'm not sure about this change. I only took a quick look, but it seems like the time would now always be the same for all loggers, which is probably not what we want.

Isn't it enough to keep the time as a member variable?

Unrelated: please also wrap the commit message so that it stays under 72 chars long (see [CONTRIBUTING.md](https://github.com/prplfoundation/prplMesh/blob/master/CONTRIBUTING.md)).

Reply:
Indeed I wanted to use the same time stamp for all the loggers. (It would be clearly visible that a bunch of *.log files belong to a single run; it may help to examine the logs with tools like `find` and `grep`).
But if it is not what we want - yes, using a member variable would be preferable.